### ISSUE TITLE
High memory demand

### ISSUE THREAD

**Issue by philippmaa** [MichalGeyer/plug-and-play#16]
Hi there, thanks for sharing this code. I am currently trying to make it work using an NVIDIA GeForce RTX 3060 with 12GB RAM. If I run "run_features_extraction.py" the call "z_enc, _ = sampler.encode_ddim(...)" finishes. However, right afterward when calling "samples_ddim, _ = sampler.sample()" I run into a "RuntimeError: CUDA out of memory" error. Is there some problem or does the model really need that much memory?
I would appreciate any help.

**Comment by chunyu-li** [comment]
Yes, it's the problem of your VRAM size. I have successfully run the code and it requires about 17G VRAM.

<img width="634" alt="Screenshot 2023-07-29 at 22 19 21" src="https://github.com/MichalGeyer/plug-and-play/assets/61857563/37d8e068-1da0-487c-b8e0-a3873a4b43d3">
